frontend/apps/app/src/trigger/jobs.ts

```diff
@@ -26,20 +25,17 @@ export const savePullRequestTask = task({
         owner: payload.owner,
         name: payload.name,
         repositoryId: payload.repositoryId,
-      } as SavePullRequestPayload)
+      })
       logger.info('Successfully saved PR to database:', { prId: result.prId })
 
-      const pullRequest = await getPullRequest(
-        payload.repositoryId,
```

Comment:
this `repositoryId` resitoryId represents `GitHub's number` . not our supabase table's one.